server/portal/apps/intromessages/intro_unit_test.py

```diff
@@ -0,0 +1,79 @@
+from django.http import JsonResponse
+from portal.apps.intromessages.views import IntroMessagesView
+import json
+import pytest
+
+from portal.apps.intromessages.models import IntroMessages
+
+
+@pytest.fixture
+def intromessage_mock(authenticated_user):
+    IntroMessages.objects.create(user=authenticated_user, component="HISTORY", unread=False)
+
+
+"""
+Test get of "read" (not unread) IntroMessages for an authenticated user
+"""
+@pytest.mark.django_db(transaction=True, reset_sequences=True)
+def test_intromessages_get(client, authenticated_user):
+    response = client.get('/api/intromessages/')
+    assert response.status_code == 200
```

Comment:
consider combining this test with `test_response_data` and call it `test_intro_messages_get`?